### PR TITLE
[DE] Cleanup `Rules`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
Cleans up this article a bit.
- Fixes a link
- Removes grammar mistakes
- Changes some words, so there's more consistency throughout the wiki

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Rules/de.md [hunk 1]
```diff
@@ -1,30 +1,28 @@
 # Regeln
 
-Diese Regeln bilden die Grundlage, mit der wir dafür sorgen, dass die osu! Community Spaß haben kann und eine freundliche Umgebung hat.
+Diese Regeln bilden die Grundlage, mit der wir dafür sorgen, dass die osu!-Community Spaß haben kann und eine freundliche Umgebung hat.
 
 ## Communityregeln
```
Comment (respektive) [inline wiki/Rules/de.md:5]:
```suggestion
## Community-Regeln
```

File: wiki/Rules/de.md [hunk 2]
```diff
@@ -1,30 +1,28 @@
...
-2. **Spiele fair** Die Benutzung von Drittanbieter-Software jeglicher Art geht nicht in Ordnung. Dazu gehören Makros, Aim-Assits, Speedhacks etc. Sobald dir ein Programm dabei hilft, etwas zu tun, was du selbst tun musst ist es nicht okay.
-3. **Seid freundlich zueinander.** Die osu! Community bietet keinen Platz für Feindseligkeiten oder Belästigungen. Wir sind alle hier um Kreise zu klicken, Trommeln zu schlagen, Früchte zu fangen oder bis zu achtzehn Tasten gleichzeitig zu drücken - und nicht um Arschlöcher zueinander zu sein.
+1. **Jeder Spieler darf zu jedem Zeitpunkt nur *EINEN* Account besitzen.** Der *erste* Account, den du während der Registrierung erstellst, ist von dem Zeitpunkt der Erstellung dein einziger osu!-Account und bleibt auch dein einziger. Er gehört niemand anderem - nicht deinem Bruder, nicht deiner Mutter, nicht deiner Schwester oder deinem Freund - nur DIR. Teile ihn mit niemandem. Da du nicht die Chance erhälst, einen neuen Account zu erstellen, solltest du also gut darauf aufpassen.
```
Comment (respektive) [inline wiki/Rules/de.md:7]:
```suggestion
1. **Jeder Spieler darf zu jedem Zeitpunkt nur *EINEN* Account besitzen.** Der *erste* Account, den du während der Registrierung erstellst, ist von dem Zeitpunkt der Erstellung dein einziger osu!-Account und bleibt auch dein einziger. Er gehört niemand anderem - nicht deinem Bruder, nicht deiner Mutter, nicht deiner Schwester oder deinem Freund - nur DIR. Teile ihn mit niemandem. Da du nicht die Chance erhältst, einen neuen Account zu erstellen, solltest du also gut darauf aufpassen.
```

File: wiki/Rules/de.md [hunk 3]
```diff
@@ -1,30 +1,28 @@
...
+2. **Überlass Problemen im Chat den Moderatoren.** Du kannst sie jedoch auf Probleme hinweisen, indem du den Befehl `!report` verwendest. Den Rest übernehmen sie dann.
+3. **Wenn jemand stumm geschaltet oder eingeschränkt wird, lass sie in Ruhe und rede nicht über sie.** Wenn jemand eingeschränkt wird, geht der Fall nur ihn und die Administratoren etwas an - und sonst niemanden!
+4. **Fördere faires Spielen und schenke Cheatern und anderen böswilligen Nutzern keine Beachtung.** Leider gibt es immer solche Leute und Aufmerksamkeit ist oft genau das, was sie weitermachen lässt oder andere zum Mitmachen anstiftet. Melde diese Nutzer mit dem Befehl `!report` und widme dich wieder anderen Dingen.
 5. **Nimm Rücksicht auf jüngere Mitspieler, die Teil dieser Community sind. Vermeide nicht jugendfreie Inhalte so weit wie möglich.** Nicht jugendfreie Dinge gehören einfach nicht in eine Community, die alle Altersgruppen umfasst. In privaten Chats kannst du dich unterhalten, wie du möchtet (übertreibe es nicht), halte dich aber in den öffentlichen Kanälen zurück.
```
Comment (respektive) [inline wiki/Rules/de.md:21]:
```suggestion
5. **Nimm Rücksicht auf jüngere Mitspieler, die Teil dieser Community sind. Vermeide nicht jugendfreie Inhalte so weit wie möglich.** Nicht jugendfreie Dinge gehören einfach nicht in eine Community, die alle Altersgruppen umfasst. In privaten Chats kannst du dich unterhalten, wie du möchtest (übertreibe es nicht), halte dich aber in den öffentlichen Kanälen zurück.
```

File: wiki/Rules/de.md [hunk 4]
```diff
@@ -1,30 +1,28 @@
...
-
-*[Moderatoren](/wiki/People/The_Team/Global_Moderation_Team) müssen sich nicht an diese Regel halten, wenn sie ihre Arbeit erledigen.*
+6. **Achte darauf, welche Inhalte du mit anderen Leuten teilst, besonders bei möglicherweise verstörenden und expliziten Darstellungen.** Wenn du zweimal darüber nachdenken musst, ob es okay ist, diesen Link zu teilen, ist er es wahrscheinlich nicht. Vermeide Inhalte mit schnell pulsierenden Effekten, Gewaltdarstellungen, Missbrauch oder anderen expliziten Inhalten.
+7. **Links sollen nur auf legale Inhalte verweisen.** Links zu beispielsweise Webseiten für Anime-Streaming oder Webseiten zu urheberrechtlich geschützten Inhalten jeglicher Natur sind nicht erlaubt, außer die Webseiten besitzen die nötige Lizenz zur Verbreitung (beispielsweise Animelab oder Crunchyroll). Du kannst Twitch-Kanäle über andere Spiele teilen, solange das in `#videogames` passiert.
+8. **Wirb nicht für Chat-Server anderer Platformen jeglicher Art, einschließlich Discord, Teamspeak oder Skype.** Im osu!-Chat geht es um osu! und nicht, um deine Server zu füllen. Nutze private Nachrichten, wenn es unbedingt sein muss.
```
Comment (respektive) [inline wiki/Rules/de.md:24]:
```suggestion
8. **Wirb nicht für Chat-Server anderer Plattformen jeglicher Art, einschließlich Discord, Teamspeak oder Skype.** Im osu!-Chat geht es um osu! und nicht, um deine Server zu füllen. Nutze private Nachrichten, wenn es unbedingt sein muss.
```

**Review by respektive — CHANGES_REQUESTED** [review]

File: wiki/Rules/de.md (hunk 1 above)
Comment (The-Last-Cookie) [reply wiki/Rules/de.md]: yeah, seems better tbh.

File: wiki/Rules/de.md (hunk 4 above)
Comment (The-Last-Cookie) [reply wiki/Rules/de.md]: It is so annoying, that this word has different writings in German in English. The same with `packets`/`Pakete`. I always miss these ~~"~~
Good that you spotted it!

Comment (respektive) [reply wiki/Rules/de.md]: yeah, really confusing sometimes

**Comment by respektive** [comment]
On line 33 `Wenn möglich unter einem halbem Megabyte (500 Kilobytes).` should be `Wenn möglich unter einem halben Megabyte (500 Kilobytes).`
Can't add comments on unchanged lines (yet) so this has to do. xd

**Review by respektive — APPROVED** [review]
👍

**Review by TicClick — APPROVED** [review]

**Pull request merged**